network/codec/roundTripHeader_test.go

```diff
@@ -0,0 +1,40 @@
+package codec_test
+
+// pushd network ; BINSTAT_LEN_WHAT="~net=99;~lock=99;~Backend=99" BINSTAT_ENABLE=1 GO111MODULE=on go test -v -coverprofile=coverage.txt -covermode=atomic --tags relic ./codec/roundTripHeader_test.go ; popd
```

Comment:
do we need this?

Reply:
@yhassanzadeh13 thanks for the comment. Removed this comment as requested! :-)